src/helper/testing/index.ts

```diff
@@ -7,13 +7,13 @@ import { hc } from '../../client'
 import type { Client } from '../../client/types'
 import type { ExecutionContext } from '../../context'
 import type { Hono } from '../../hono'
+import type { Env, Schema } from '../../types'
```

Comment:
Can you remove `Env` to avoid the linter warning?

Reply:
Sorry, I forgot it. wait a moment...